app/controllers/mailchimp/configurations_controller.rb

```diff
@@ -6,6 +6,22 @@ class Mailchimp::ConfigurationsController < Mailchimp::PetalController
   
   def show
     # @configuration setted by get_configuration
+    if @configuration.api_key
+      params[:app_key] = ENV["contacts_key"]
+      params[:api_key] = @configuration.api_key
+      response = Typhoeus.get("#{Contacts::HOST}/v0/mailchimp_synchronizers/#{@configuration.api_key}", params: params)
+
+      if response.success?
+        status = JSON.parse(response.body)["status"]
+        if status == "working"
+          flash.now[:notice] = t('mailchimp.list.status.working')
+        elsif status == "failed"
+          flash.now[:alert] = t('mailchimp.list.status.failed')
+        end
+      else
+        flash.now[:alert] = t('mailchimp.list.status.not_found')
```

Comment:
Si está en :ready no imprime ningún mensaje. Va al not found sólo si responde un 404, o un "not sucess". 
En el caso del ready entra en el primer IF, pero ahí no matchea en ninguno de los de adentro.